libs/partners/ai21/langchain_ai21/llms.py

```diff
@@ -0,0 +1,141 @@
+import asyncio
+from functools import partial
+from typing import (
+    Any,
+    List,
+    Optional,
+)
+
+from ai21.models import CompletionsResponse, Penalty
+from langchain_core.callbacks import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain_core.language_models import BaseLLM
+from langchain_core.outputs import Generation, LLMResult
+
+from langchain_ai21.ai21_base import AI21Base
+
+
+class AI21(BaseLLM, AI21Base):
```

Comment:
Thoughts on making this `AI21LLM` instead? Clearer for ai21 as a provider that offers multiple types!

Reply:
Fixed